packages/zoe/src/contracts/simpleExchange.js

```diff
@@ -1,7 +1,7 @@
 import harden from '@agoric/harden';
-// Eventually will be importable from '@agoric/zoe-contract-support'
-import { producePromise } from '@agoric/produce-promise';
 import { makeZoeHelpers, defaultAcceptanceMsg } from '../contractSupport';
+// TODO update to '@agoric/notifier' ASAP
```

Comment:
I think if the packages are set up correctly in our workspaces, you can use `@agoric/notifier` already, so we shouldn't need this comment and the local file import.

Reply:
That would have been nice, but yarn install reports
```
error Couldn't find package "@agoric/notifier@^0.0.1" required by "@agoric/zoe@0.5.0" on the "npm" registry.
```
which makes it sound like it has to be released and registered before it can be imported.

Reply:
Maybe the right answer is to fix this immediately after merging the notifier PR, and before its own merge. That will work.

Reply:
It does sound like that, but if you add the packages to the workspaces, it should fix that error

Reply:
`yarn add @agoric/notifier@0.0.1` reports
```
error An unexpected error occurred: "https://registry.yarnpkg.com/@agoric%2fnotifier: Not found".
```
implying that it wants to find it in a registry. If I add `@agoric/notifier` to the imports manually, tests fail, and `yarn install` complains.

What do you mean by "add the packages to the workspaces" that's different from calling `yarn install`?

Reply:
There's a workspaces property in the root package.json:

![Screen Shot 2020-04-16 at 1 57 54 PM](https://user-images.githubusercontent.com/2441069/79506149-57c7ad00-7fea-11ea-928a-c63528a1aaba.png)

If you add it there, and the package is set up properly, the imports should work, I think.

Reply:
Thanks. That worked.